src/foundry/client/apps/sidebar/apps/tours-management.d.mts

```diff
@@ -18,23 +18,20 @@ declare global {
      */
     static override get defaultOptions(): PackageConfiguration.Options;
 
-    override _prepareCategoryData(): ToursManagement.TourCategory;
+    override _prepareCategoryData(): PackageConfiguration.CategoryData;
 
     override activateListeners(html: JQuery<HTMLElement>): void;
 
+    protected override _onResetDefaults(event: JQuery.ClickEvent): void;
+
     /**
      * Handle Control clicks
      * @internal
      */
-    protected _onClickControl(event: JQuery.ClickEvent): void;
+    protected _onClickControl(event: JQuery.ClickEvent): MaybePromise<unknown>;
   }
 
   namespace ToursManagement {
-    interface TourCategory extends PackageConfiguration.Category {
```

Comment:
Is there a reason you deleted this? My reading of the code suggests this is still valid

Reply:
Yeah, brain fart. When I switched over `_prepareCategoryData()` to return `PackageConfiguration.CategoryData` instead of `ToursManagement`TourCategory` I also went to remove this because it was no longer used anywhere. Still a valid interface though. In fact, should I also define a `TourCategoryData` which extends `PackageConfiguration.CategoryData` but uses `TourCategory[]` as the type for `categories` instead of `Category[]`, then use that as the return type for `_prepareCategoryData()`?

Reply:
Yes, that seems to be the most helpful/correct strategy here.